src/thread_timer.h

```diff
@@ -9,13 +9,14 @@ namespace internal {
 
 class ThreadTimer {
  public:
-  ThreadTimer() = default;
+  ThreadTimer(bool measure_process_cpu_time_)
```

Comment:
https://abseil.io/tips/94, which discusses the non-readability of callsites when using boolean parameters.

Consider two factory methods (`Create()` and `CreateProcessCpuTime()`) that can hide the boolean. The enum example in the above absl tip is likely overkill for this.